pkg/registry/resource/resourceclaimtemplate/strategy_test.go

```diff
@@ -179,14 +180,14 @@ var testCapacity = map[resource.QualifiedName]apiresource.Quantity{
 
 var objWithCapacityRequests = func() *resource.ResourceClaimTemplate {
 	obj := obj.DeepCopy()
-	addSpecDeviceRequestWithCapacityRequests(obj, testCapacity, false)
+	addSpecDeviceRequestWithCapacityRequests(obj, testCapacity, false, true)
 	return obj
 }()
 
 func addSpecDeviceRequestWithCapacityRequests(resourceClaimTemplate *resource.ResourceClaimTemplate,
-	capacity map[resource.QualifiedName]apiresource.Quantity, prioritizedListFeature bool) {
+	capacity map[resource.QualifiedName]apiresource.Quantity, prioritizedListFeature, withDistinctAttribute bool) {
 	r := resource.DeviceRequest{
-		Name: "req-0",
+		Name: "cap-req-0",
```

Comment:
It seems like setting `prioritizedListFeature` to `true` here will create a ResourceClaim with both `Exactly` and `FirstAvailable` set. That is not allowed, so would be cleaner if we only added `Exactly` when `prioritizedListFeature` is `false`

Reply:
Sure, I have updated. Thank you.